api/desecapi/debug.py

```diff
@@ -0,0 +1,12 @@
+from django.views.debug import ExceptionReporter
+
+
+class PayloadExceptionReporter(ExceptionReporter):
+    def get_traceback_data(self):
+        data = super().get_traceback_data()
+        if self.request is not None:
+            try:
+                data["request_meta"]["_body"] = self.request.body
+            except:
```

Comment:
catching all exceptions is bad practice because it can hide problems and make debugging hard - what is the reason you want to do it here anyways?

Reply:
`request.body` is a property that either reads from a cached `._body` attribute or from a stream. Indeed, who knows what could go wrong.

On the other hand, we're already in error handling code for a *different* error. We just want to cleanly log it and then move away from it with a sane mind. I'm not sure if we should go into second-order error-handling issues.

Further, any issue in that `try` code is not our issue, but Django's. I'm not sure we could enumerate all exception types that potentially could occur, even more so should their code change.

Errors in `request.body` will be properly handled whenever it is used "normally" (outside of error handling) during request handling; they would then be a direct cause of exception handling and cause full logging (with traceback etc.).

Reply:
Agree that its not realistic to draft and maintain a list of exceptions. What's more, upstream code for `ExceptionReporter` uses similar except clauses to determine other fields of the report.

So, I don't like this solution but also don't have a better way of doing it. We do need the body in the reports.